CHANGELOG.md

```diff
@@ -201,7 +201,7 @@ under the License.
 - [#16118](https://github.com/apache/superset/pull/16118) fix: change Alert Permissions (@AAfghahi)
 - [#16088](https://github.com/apache/superset/pull/16088) fix(explore): dnd error when dragging metric if multi: false (@kgabryje)
 - [#16092](https://github.com/apache/superset/pull/16092) fix(Explore): Adjust width of cell in Time-Series Table chart (@geido)
-- [#16132](https://github.com/apache/superset/pull/16132) fix: virtual dataset wont work (@zhaoyongjie)
+- [#16132](https://github.com/apache/superset/pull/16132) fix: virtual dataset won't work (@zhaoyongjie)
```

Comment:
The changelog entries should probably not be altered, as these correspond to the actual PR titles.

Reply:
Makes sense - I have updated the PR!